Slambook/index.js

```diff
@@ -54,30 +55,8 @@ form.addEventListener("submit", (e) => {
```

Comment:
To set classes for an element, use 'element.classList.add('card')' instead of directly assigning 'element.classList'.